tasks/main.yml

```diff
@@ -34,6 +34,34 @@
       when: |
         __certificate_provider_vars[__certificate_provider].packages is defined
 
+    - name: Ensure pre-scripts hooks directory exists
```

Comment:
The permissions and ownership of the script directories needs to be locked down.  I would suggest the certmonger user/group for the directory user/group, and `0700` for directory permission, and `0700` for file permission.

Reply:
Ok. I'll create variables for that.